README.md

```diff
@@ -83,6 +83,16 @@ php composer.phar install
 ```
 Debian/Ubuntu people, just execute `install_ubuntu.sh`
 
+## Nightly Builds
```

Comment:
This needs to be a h3 (three ###), otherwise the »Resetting the app« down below is a sub-point of this here which doesn’t make sense.